src/client/sandbox/node/element/style.js

```diff
@@ -0,0 +1,220 @@
+import SandboxBase from '../../base';
+import { overrideDescriptor, createOverriddenDescriptor } from '../../../utils/property-overriding';
+import styleProcessor from '../../../../processing/style';
+import { getProxyUrl, parseProxyUrl } from '../../../utils/url';
+
+const CSS_STYLE_IS_PROCESSED = 'hammerhead|style|is-processed';
+const CSS_STYLE_PROXY_OBJECT = 'hammerhead|style|proxy-object';
+const CSS_STYLE_PROXY_TARGET = 'hammerhead|style|proxy-target';
+
+export default class StyleSandbox extends SandboxBase {
+    constructor () {
+        super();
+
+        this.URL_PROPS        = ['background', 'backgroundImage', 'borderImage',
+            'borderImageSource', 'listStyle', 'listStyleImage', 'cursor'];
+        this.DASHED_URL_PROPS = StyleSandbox._generateDashedProps(this.URL_PROPS);
+
+        // NOTE: The CSS2Properties class is supported only in the Firefox
+        // and its prototype contains all property descriptors
+        this.isProtoContainsAllProps = !!window.CSS2Properties;
```

Comment:
1)Declare boolean props without `is` suffix.

2)These flags are feature detection related only to styles. I propose to combine them to the single variable.
```js
constructor () {
   ....
   StyleSandbox._detectBrowserFeatures (); // or just '._detectFeatures'
 
and inside this function fill the variable:

this.features = {
    protoContainsAllProps:...,
    protoContainsUrlDescriptors: ....
    ....
};
  ...
}
```